src/styles/home_footer.css

```diff
@@ -0,0 +1,47 @@
+.home-footer-nav-container {
+  padding-left: 40px;
+  padding-right: 40px;
+  background-color: #3d4351;
+}
+
+@media only screen and (max-width: 475px) {
+    /* For mobile phones: */
+    [class*="home-footer-nav"] {
```

Comment:
Why are we using substring matching of the class name instead of a class selector? Indentation off